caliban/main.py

```diff
@@ -36,142 +22,103 @@
 import caliban.platform.shell as ps
 import caliban.util.schema as cs
 
+# Set logging level for main logger
 ll.getLogger("caliban.main").setLevel(logging.ERROR)
 t = Terminal()
 
 
 def run_app(arg_input):
-  """Main function to run the Caliban app. Accepts a Namespace-type output of an
```

Comment:
can you please install the `pre-commit` hooks? 

```
pip install pre-commit
pre-commit --install
```

and then

```
pre-commit run --all-files
```

if you want to trigger them manually. This will clear up all formatting differences so we can look at what's actually changed. Thank you!